modular_robot/revolve2/modular_robot/body/v1/_brick_v1.py

```diff
@@ -0,0 +1,23 @@
+from pyrr import Vector3
+
+from .._color import Color
+from .._right_angles import RightAngles
+from ..common import Brick
+
+
+class BrickV1(Brick):
+    """A brick module for a modular robot."""
+
+    BOUNDING_BOX = Vector3([0.06288625, 0.06288625, 0.0603])
+    MASS = 0.030
+    CHILD_OFFSET = 0.06288625 / 2.0
+    color = Color(50, 50, 255, 255)
+    num_children = 3
+
+    def __init__(self, rotation: float | RightAngles):
+        """
+        Initialize this object.
+
+        :param rotation: The modules rotation.
+        """
+        super().__init__(3, rotation, self.color)
```

Comment:
continuing on what I said in the base brick class. Let's pass the actual values of the mass, child offset, etc to this constructor. We can keep the MASS and CHILD_OFFSET etc constants, but make them private and directly pass them to the parent constructor and don't actually use them anywhere else than when putting them in the constructor. so we use the variables defined by the parent. When creating a new brickv3 of something at some point or whatnot we cannot forget to set the values.